### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by eggrobin**
And update the Mathematica-generating code.

### CODE REVIEW

File: integrators/embedded_explicit_runge_kutta_nyström_integrator_test.cpp [hunk 1]
```diff
@@ -182,14 +173,13 @@ TEST_F(EmbeddedExplicitRungeKuttaNyströmIntegratorTest,
...
   std::int64_t const steps_forward = 132;
 
-  int evaluations = 0;
   auto const step_size_callback = [](bool tolerable) {};
 
   std::vector<ODE::SystemState> solution;
   ODE harmonic_oscillator;
   harmonic_oscillator.compute_acceleration =
       std::bind(ComputeHarmonicOscillatorAcceleration,
-                _1, _2, _3, evaluations);
```
Comment (pleroy) [inline integrators/embedded_explicit_runge_kutta_nyström_integrator_test.cpp]: Comment the `nullptr`.

File: mathematica/integrator_plots.cpp [hunk 2]
```diff
@@ -214,262 +251,114 @@ void GenerateSimpleHarmonicMotionWorkErrorGraphs() {
...
 
-void GenerateKeplerProblemWorkErrorGraphs() {
-  SRKNIntegrator::Parameters<Length, Speed> parameters;
-  SRKNIntegrator::Solution<Length, Speed> solution;
-  // Semi-major axis.
-  Length const a = 0.5 * Metre;
-  // Velocity.
-  Speed const v = 0.5 * Metre / Second;
-  // Gravitational parameter of the system, μ = G(m + m).
+void GenerateKeplerProblemWorkErrorGraphs(double eccentricity) {
```
Comment (pleroy) [inline mathematica/integrator_plots.cpp]: `double const`

Comment (eggrobin) [inline mathematica/integrator_plots.cpp]: Done.

File: integrators/embedded_explicit_runge_kutta_nyström_integrator_test.cpp (hunk 1 above)
Comment (eggrobin) [inline integrators/embedded_explicit_runge_kutta_nyström_integrator_test.cpp]: Done.

**Pull request merged**